topics/metagenomics/tutorials/taxonomic-profiling/tutorial.md

```diff
@@ -35,27 +36,31 @@ The term **"microbiome"** describes "a characteristic microbial community occupy
+For metagenomic data analysis we start with sequences derived from DNA fragments that are isolated from the sample of interest. Ideally, the sequences from all microbes in the sample are present. The underlying idea of taxonomic assignment is to compare the DNA sequences found in the sample (reads) to DNA sequences of a database. When a read matches a database DNA sequence of a known microbe, we can derive a list with microbes present in the sample.
 
 When talking about taxonomic assignment or taxonomic classification, most of the time we actually talk about two methods, that in practice are often used interchangeably:
 - **taxonomic binning**: the clustering of individual sequence reads based on similarities criteria and assignation of clusters to reference taxa
 - **taxonomic profiling**: classification of individual reads to reference taxa to extract the relative abundances of the different taxa
 
+The reads can be obtained from **amplicon sequencing**  (e.g. 16S, 18S, ITS), where only specific gene or gene fragments are targeted (using specific primers) or **shotgun metagenomic sequencing**, where all the accessible DNA of a mixed community is amplified (using random primers). 
+This tutorial focuses on reads obtained from shotgun metagenomic sequencing. 
+
 ## Taxonomic profiling
 
 Tools for taxonomic profiling can be divided into three groups. Nevertheless, all of them require a pre-computed database based on previously sequenced microbial DNA or protein sequences.
-1. **DNA-to-DNA**: comparison of sequencing reads with genomic databases of DNA sequences, with tools like Kraken ({% cite Wood2014 %})
-2. **DNA-to-Protein** : compare sequencing reads with protein databases (more computationally intensive because of analysis of all six frames of potential DNA-to amino acid translation, with tools like DIAMOND
-3. **Marker based**: search for marker genes (e.g. 16S rRNA sequence) in reads, which is quick, but introduces bias, with tools like MetaPhlAn ({% cite blanco2023extending %})
+1. **DNA-to-DNA**: comparison of sequencing reads with genomic databases of DNA sequences with tools like Kraken ({% cite Wood2014 %})
+2. **DNA-to-Protein** : comparison of sequencing reads with protein databases (more computationally intensive because all six frames of potential DNA-to amino acid translations need to be analyzed) with tools like DIAMOND)
+3. **Marker based**: searching for marker genes (e.g. 16S rRNA sequence) in reads, which is quick, but introduces bias, with tools like MetaPhlAn ({% cite blanco2023extending %})
+4. **Mixed**: Combined search using e.g. maker genes and translated searches, with tools like HUMAnN ({% cite franzosa_species-level_2018 %})
```

Comment:
Not sure about that. HUMAnN does not do taxonomic profiling, it relies completely on MetaPhlAn for that

Reply:
Yes, that was a brain fart from me, it does indeed use MetaPhlAn and diamond for translated search, but does not traxonomic profile the found proteins ... but when I think of an example of **DNA-to-Protein** for taxonomy, can diamond also assing taxonomy ? Whichever, I remove that part.